.github/workflows/build.yml

```diff
@@ -0,0 +1,55 @@
+name: Index and build site
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  schedule:
+    - cron:  "0 */2 * * *"
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  index:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Setup Pages
+      uses: actions/configure-pages@v5
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 12
```

Comment:
It was the latest version of Node.js that could run the statusboard.

Reply:
Yeah this was written a long time ago. We should spend some cycles getting that fixed in [the upstream](https://github.com/pkgjs/statusboard).